src/components/EditHospitalForm/index.js

```diff
@@ -96,6 +100,28 @@ const EditHospitalForm = ({ errors, setErrors, hospital }) => {
           />
         </FormGroup>
 
+        <FormGroup>
+          <Label htmlFor="hospital-survey-url" className="nhsuk-label--l">
+            Key contact survey URL
```

Comment:
Could we add `(optional)` to make it clear to the user that this isn't required?

```suggestion
            Key contact survey URL (optional)
```

This is the suggestion way by the NHS Design System. See https://service-manual.nhs.uk/content/how-to-write-good-questions-for-forms/make-sure-you-need-each-question#start-with-essential-questions.

Reply:
Ah thanks I was wondering how to display the optionality 🙌